package.json

```diff
@@ -30,8 +30,8 @@
     "url": "https://github.com/andig/pimatic-fritz"
   },
   "dependencies": {
-    "smartfritz-promise": "^0.4.1",
```

Comment:
Good question. If we merge this it may force an update, likely to 0.5 which is compatible. With the 0.5 problems you've seen- is this safe? Open for suggestion here. If unsafe rather revert this requirement change for now. In any case the version number needs a bump ;)